src/discord/utils/tripsitme.ts

```diff
@@ -130,7 +130,7 @@ export async function needsHelpmode(
   // log.debug(`[${PREFIX}] targetHasRoleNeedshelp: ${targetHasRoleNeedshelp}`);
 
   // Save the user's roles to the DB
-  const targetRoleIds = target.roles.cache.map(role => role.id);
+  const targetRoleIds = target.roles.cache.map((role: { id: any; }) => role.id);
```

Comment:
@spr1t3-0 Why make this change?

Reply:
it was some kind of auto quick-fix thing from vscode